docs/regex/approximate.md

```diff
@@ -35,60 +35,63 @@ The surplus `e` in `neet` is deleted.
 Similarly:
 
 - `doubleclick.net` is matched by `^(doubleclicky\.netty){-3}$`
-- `doubleclick.net` is NOT matched by `^(doubleclicky\.nettfy){-3}$`
+- `doubleclick.net` is **not** matched by `^(doubleclicky\.nettfy){-3}$`
 
 ## Accepted **substitutions** (`#`)
 
-Use `(something){#x}` to specify that the regex should still be matching when `x` characters would need to be *substituted* from the sub-expression `something`:
+Use `(something){#x}` to specify that the regex should still match when `x` characters would need to be *substituted* in the sub-expression `something`:
 
 Example 1:
 
-- `oobargoobaploowap` is matched by `(foobar){#2~2}`
-Hint: `goobap` is `foobar` with two substitutions `f->g` and `r->p`
+- `oobargoobaploowap` is matched by `(foobar){#2}`
```

Comment:
Both `#2` and `#2~2` are valid.

I think the `~2` was intentional here.

Reply:
I thought it better not to use the `~` syntax before the section explaining how it works